tds-prize-tool/src/utils.rs

```diff
@@ -0,0 +1,3 @@
+pub(crate) fn sol_to_lamports(sol: f64) -> u64 {
```

Comment:
We should sneak this into sdk/ somewhere.

Reply:
Sounds good: https://github.com/solana-labs/solana/pull/6192